narwhals/typing.py

```diff
@@ -43,7 +43,7 @@ def __dataframe__(self, *args: Any, **kwargs: Any) -> Any: ...
 ]
 """Anything which can be converted to a Narwhals DataFrame or LazyFrame."""
 
-Frame: TypeAlias = Union["DataFrame[Any]", "LazyFrame[Any]"]
+Frame: TypeAlias = Union["IntoFrame", "IntoDataFrame"]
```

Comment:
Not sure if this is appreciate, or should we create it on top of `dataframe.py`?

Reply:
not sure I follow, sorry, what's the issue with the current definition of `Frame` here to mean "either a Narwhals DataFrame, or a Narwhals LazyFrame"?